iso690-numeric-brackets-cs.csl

```diff
@@ -420,8 +420,15 @@
       <key variable="citation-number"/>
     </sort>
     <layout prefix="[" suffix="]" delimiter=", ">
-      <text variable="citation-number"/>
-      <text variable="locator" prefix=" s. "/>
+      <group delimiter=", ">
+        <group delimiter=" ">
+          <text variable="citation-number"/>
+        </group>
+        <group delimiter=" ">
+          <label variable="locator" form="short"/>
+          <text variable="locator"/>
+        </group>
+      </group>
     </layout>
```

Comment:
You are right about the outer group, but the one-child-element-group seems unnecessary. If you want to have a space between `citation-number` and the other two variables, then how about:
```
<layout prefix="[" suffix="]" delimiter=", ">
  <group delimiter=" ">
    <text variable="citation-number"/>
    <group delimiter=" ">
      <label variable="locator" form="short"/>
      <text variable="locator"/>
    </group>
  </group>
</layout>
```